### DISCUSSION TITLE
Jean-Thierry Kéchichian Community Award 2024 - nominate exemplary community members here!

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16553] [General]
After the passing of our beloved [Jean-Thierry](https://docs.orchardcore.net/en/latest/community/jean-thierry/) to honor him, keep his memory alive, and foremost, inspire people to do good work for the community, we launched [an award named after him](https://docs.orchardcore.net/en/latest/community/jean-thierry-community-award/) for exceptional community members. Every year, the community votes on who has done the most for Orchard Core. 

**It's your turn to nominate a community member!** In a message below, add somebody (`@mention` them) who you think has done most for Orchard Core, with a sentence explaining what they've done. This can entail activities in the form of e.g. code contributions (including documentation), help under issues and discussion, evangelization outreach, community management, or anything else. However, it needs to be public and tangible.

**Nominations are open until the 27th midnight, Anywhere on Earth.** After that, we'll vote on the nominees secretly. The winner is then announced at the [Orchard Harvest conference](https://orchardcore.net/harvest) on the 12th of September.

**What does the winner get?**

The award is foremost about recognition. However, the community provides some perks too:

- Mention of each year's winner [in the docs](https://docs.orchardcore.net/en/latest/community/jean-thierry-community-award/#winners-of-the-jean-thierry-kechichian-community-award).
- Post about it under the Orchard Core social media.
- [Lombiq](https://lombiq.com/) provides free custom domain usage on [DotNest](https://dotnest.com/), publicizing the winner in Lombiq's media, and a year of sponsorship similar to [other open source contributors and projects](https://lombiq.com/open-source).

If your company works with Orchard Core and would like to incentivize community members, please let us know what else you can add!

**Comment by MikeAlhayek** [comment]
@Piedone 
@sebastienros 
@agriffard 
@gvkries

**Comment by agriffard** [comment]
@deanmarcussen
@hishamco  
@MikeAlhayek 
@ns8482e 
@Piedone
@Skrypt

**Comment by sebastienros** [comment]
@jtkech

**Reply by Piedone** [reply]
While the best would be not to have this award due to JT still living, this is now for those who he inspired, and continue to inspire.

**Reply by hishamco** [reply]
We might rename the award to Orchard Community Award and JT is the winner this year, the gift could goes to his family if we do so

**Comment by hishamco** [comment]
IMHO we could consider the contribution not in the Orchard Core repo, but it might be extended to any Orchard Core repos or OC Community repositories

**Reply by Piedone** [reply]
This is already the case :).

**Reply by hishamco** [reply]
I thought it's only OC repo contributions :)

**Comment by Piedone** [comment]
I nominate @MikeAlhayek for the huge amount of improvements he contributed to the codebase.

**Comment by gvkries** [comment]
I also nominate @MikeAlhayek for his daily contributions. Is there even a single day he does not add something? 😉

**Comment by hishamco** [comment]
We might have certain criteria to choose the winner, I remember @deanmarcussen made a huge contribution a long time ago, also @Skrypt and lately @MikeAlhayek

So almost all guys deserve a prize :) for that, I'm still nominating JT for the tremendous amount of contribution

**Reply by Piedone** [reply]
To clarify what I've written under https://github.com/OrchardCMS/OrchardCore/discussions/16553#discussioncomment-10330059: JT is not with us anymore, and giving him a posthumous award that we created after his passing specifically to honor him, and to inspire people _because_ he is not with us anymore, doesn't make sense. This is for people who we see as a peer of JT.

There are many people worthy of recognition, definitely, and now we should select the one for this year who even excels among them.

**Reply by hishamco** [reply]
We can consider the GitHub contributions too in Orchard Core & Orchard Core Community repos

**Reply by Piedone** [reply]
Anything that made the Orchard Core project, ecosystem, or community better!

**Comment by Skrypt** [comment]
@jtkech for all the previous years contributions.
@MikeAlhayek for this year.

I can't be nominated as I did not contribute that much this last year. 😉

**Comment by hishamco** [comment]
I will take a different way because there are other community projects than the code base

**Orchard Core**

- I vote for @jtkech and @deanmarcussen for their massive contributions
- If the nomination for this year I will vote for my friend @MikeAlhayek 

**Orchard Core Commerce**

- @sarahelsaig did great work making the [Commerce](https://github.com/OrchardCMS/OrchardCore.Commerce) module happen

**Orchard Core Contrib**

I will nominate myself :) because I'm the only active member that maintains Orchard Core Community repos such as
- [OrchardCoreContrib.PoExtractor](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor)
- [OrchardCoreContrib](https://github.com/OrchardCoreContrib/OrchardCoreContrib)
- [OrchardCoreContrib.Modules](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules)
- [OrchardCoreContrib.Themes](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Themes)
- [OrchardCoreContrib.Testing](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Testing)

Alongside to my contribution to both OrchardCore & OrchardCore.Translations

**Reply by hishamco** [reply]
Of course, Lombiq has many OSS, I'm not sure if there's special one to be nominated

**Comment by Piedone** [comment]
Voting is open! https://github.com/OrchardCMS/OrchardCore/discussions/16632
